src/main/steps/applicant/support-you-need-during-case/summary/content.test.ts

```diff
@@ -11,24 +11,12 @@ const en = {
     aboutYou: 'About you',
   },
   keys: {
-    communicationSupportOther: 'Describe what you need',
-    courtComfort: 'I need something to make me feel comfortable during a court hearing',
-    courtHearing: 'I would need to bring support with me to a court hearing',
-    courtProceedingProvideDetails: 'Provie details',
-    describeOtherNeed: 'Describe what you need',
```

Comment:
wy these fields removed, do we not need them?

Reply:
Summary screen should be displayed based on the routing, so we have added additional conditions to it